libs/application/templates/transport-authority/license-plate-renewal/src/fields/PlateField/PlateSelectField.tsx

```
@@ -111,12 +147,9 @@ export const PlateSelectField: FC<
                   message={
                     <Box>
                       <BulletList>
-                        {!!selectedPlate.validationErrorMessages?.length &&
-                          selectedPlate.validationErrorMessages?.map(
-                            (error) => {
-                              return <Bullet>{error.defaultMessage}</Bullet>
-                            },
-                          )}
+                        {selectedPlate.validationErrorMessages?.map((error) => {
+                          return <Bullet>{error.defaultMessage}</Bullet>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)